packages/types/src/documents/global/tenantInfo.ts

```diff
@@ -0,0 +1,13 @@
+import { Document } from "../document"
+
+export interface TenantInfo extends Document {
+  owner: {
+    email: string
+    password?: string
+    ssoId?: string
+    givenName?: string
+    familyName?: string
+    budibaseUserId?: string
+  }
+  id: string
```

Comment:
What is this `id` property? How does this differ from the `_id` prop of a document?

Reply:
id is the `tenantId` - happy to rename the prop for clarification

Reply:
renamed to `tenantId`